controllers/admin/AdminCategoriesController.php

```diff
@@ -452,12 +452,12 @@ public function renderForm()
         $format = array();
         $thumb = $thumb_url = '';
         $formated_category= ImageType::getFormattedName('category');
```

Comment:
I think it would be great to rename all `formated_xx` variables to `formatted_xx` 🙏